packages/babel-plugin-proposal-decorators/test/fixtures/2021-12-methods--to-es2015/private/exec.js

```diff
@@ -1,8 +1,4 @@
 function dec(fn, context) {
```

Comment:
The original test case is renamed to `*-with-initializers/exec.js`. They are replaced by a simpler test without initializers, which exactly catches this bug.